bench/workgen/workgen.cpp

```diff
@@ -2973,7 +2973,7 @@ WorkloadOptions::WorkloadOptions()
       timestamp_advance(0.0), max_idle_table_cycle_fatal(false), create_count(0),
       create_interval(0), create_prefix(""), create_target(0), create_trigger(0), drop_count(0),
       drop_interval(0), drop_target(0), drop_trigger(0), random_table_values(false),
-      mirror_tables(false), mirror_suffix("_mirror"), _options()
+      mirror_tables(false), mirror_suffix("_mirror"), background_compact(0), _options()
```

Comment:
Can these be alphabetized? (Not just yours but earlier ones as well)

Reply:
The C++ constructor init list should be done in the same order as their declaration in the header file. It seems that we have been appending new variables to workgen.h for the `WorkloadOptions` struct without taking care of the order.

Reply:
@sueloverso, I have created WT-11481 to deal with this as it has a few impacts and would extend the scope of the current changes.